src/main/java/hu/bme/mit/spaceship/TorpedoStore.java

```diff
@@ -14,6 +14,8 @@
 
   private int torpedoCount = 0;
 
+  private Random generator=new Random();
```

Comment:
Ok, but it would be better to use SecureRandom generator. It relies on a cryptographically strong random number generator.